source/playbooks/refining-and-improving.rst

```diff
@@ -32,6 +32,17 @@ Use the run timeline to help write an accurate retrospective. Events such as own
 .. image:: ../images/Retro.gif
    :alt: Create and publish retrospective reports.
 
+Metrics
+~~~~~~~
+
+Use metrics to identify key areas in which you want to measure performance and improvement, and extract valuable insights. 
+
+These metrics can be anything that's of interest to you and your team. For example, for a software release playbook you might want to have a metric tracking how many bugs were detected during a run. The output of the metrics you've added is provided in the retrospective and, over time, you can use the data to examine anomalies and refine goals.
```

Comment:
```suggestion
These metrics can be anything that's of interest to you and your team. For example, for a software release playbook, you might want to have a metric tracking how many bugs were detected during a run. The output of the metrics you've added is provided in the retrospective and, over time, you can use the data to examine anomalies and refine goals.
```

Reply:
If you decide to move the dependency information to the top of this section, perhaps reword the last line here to: "Over time, you can use metrics across retrospectives to examine anomalies and refine goals."